pkgs/by-name/ko/koboldcpp/package.nix

```diff
@@ -106,14 +121,31 @@ effectiveStdenv.mkDerivation (finalAttrs: {
   env.NIX_LDFLAGS = lib.concatStringsSep " " (finalAttrs.darwinLdFlags ++ finalAttrs.metalLdFlags);
 
   env.NIX_CFLAGS_COMPILE =
-    lib.optionalString (march != "") "-march=${march}" + lib.optionalString (mtune != "") "-mtune=${mtune}";
+    let
+      inherit (stdenv.hostPlatform)
+        isx86_64
+        isAarch64
+        isLinux
+        isDarwin
+        ;
+    in
+    lib.concatStringsSep " " [
+      (lib.optionalString isx86_64 "-march=x86-64")
```

Comment:
I'm pretty sure this one is identical to the default behaviour either of the wrapper or of gcc/clang -> can be left out